app/models/host.rb

```diff
@@ -73,6 +83,8 @@ class Host < ApplicationRecord
   has_many :assigned_internal_profiles, -> { external(false) },
            through: :policies, source: :profiles
 
+  scope :non_edge, -> { where(HOST_TYPE.eq(nil).or(HOST_TYPE.not_eq('edge'))) }
```

Comment:
What would be resulting SQL for `HOST_TYPE.eq(nil)`?

Reply:
IS NULL